tests/Fixer/PhpUnit/PhpUnitStaticAssertionCallsFixerTest.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Fixer\PhpUnit\PhpUnitStaticAssertionCallsFixer;
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+use PHPUnit\Framework\Assert;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitStaticAssertionCallsFixer
+ */
+final class PhpUnitStaticAssertionCallsFixerTest extends AbstractFixerTestCase
+{
+    public function testFixerContainsAllPhpunitStaticMethodsInItsList()
+    {
+        $assertionRefClass = new \ReflectionClass(Assert::class);
```

Comment:
@kubawerlos I've added the test that ensures all methods of `PHPUnit\Framework\Assert` are in the list. However we can't do this for all `PHPUnit\Framework\TestCase` methods too because there are too many false positive.

Reply:
@Slamdunk `PHPUnit\Framework\TestCase` is too much indeed, also from the name of fixer I would expect only method started with "assert" to be considered.

Reply:
either drop methods from `TestCase` or remove `assert` part from the name.

what are those false positives when iterating over `TestCase` methods? could you give an example ?

Reply:
The false positive where generated by a bug in the fixer test.
I've fixed it and now I've moved all the algorithm to consider `PHPUnit\Framework\TestCase` as the unique reference.

Reply:
nice :+1: